functions/src/apis/users.ts

```diff
@@ -1,13 +1,13 @@
 import {Response as responseType} from "express";
 import {v4 as uuidv4} from "uuid";
-import {db} from "../admin";
+import {functions, db} from "../admin";
 
 // define types
 type entryType = {
     user_id: string,
     first_name: string,
     last_name: string,
-    phone_number: string, // check if auth/google profile provides phone numbers
+    email: string,
```

Comment:
I'll change the ERD to reflect this, it makes more sense to have an email than a phone number.

Reply:
Ya I was thinking even if I switch to sign in with google, I'm not sure how often the phone number field would be populated in someone's google profile, but at least email is provided (or at least I imagine so).

Reply:
Agree. Phone number is not necessary for out project scope.